backend/editor/entries.py

```diff
@@ -44,6 +49,24 @@ def create_node(self, label, entry, main_language_code):
         result = get_current_transaction().run(" ".join(query), params)
         return result
 
+    def parse_taxonomy(self, filename):
+        parser_object = parser.Parser()
+        try:
+            parser_object(filename, self.branch_name, self.taxonomy_name)
+            return True
+        except:
+            return False
+
+    def import_from_github(self):
+        base_url = "https://raw.githubusercontent.com/openfoodfacts/openfoodfacts-server/main/taxonomies/"
+        filename = ('_'.join(self.taxonomy_name.lower().split()) + '.txt')

```

Comment:
small tip, you can use tempfile module to create a tempfile (even with a context manager !)